### PR TITLE
fix: issues related to the emoji icon picker

### PR DESCRIPTION
**Pull request by asjqkkkk**
### Feature Preview

- Feat: keep the icon panel open after click ramdom
- Feat: the type of selector opened depends on the already set icon or emoji
- Feat: the skin tone of the random emoji follows the selected skin ton
- Feat: save ramdom icon in recent section
- Fix: remove the scrolling conflict of the icon picker on macOS
- Fix: the icon is not supported in sites tab

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/emoji_icon_widget.dart [hunk 1]
```diff
@@ -84,7 +84,7 @@ class RawEmojiIconWidget extends StatelessWidget {
           final iconData = IconsData.fromJson(jsonDecode(emoji.emoji));
           return IconWidget(
             data: iconData,
-            size: emojiSize,
+            size: emojiSize * 0.9,
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/emoji_icon_widget.dart:87]: Why does the emoji size need to be multiplied by 0.9?

File: frontend/appflowy_flutter/lib/shared/icon_emoji_picker/icon_search_bar.dart [hunk 2]
```diff
@@ -105,6 +110,17 @@ class _SearchTextFieldState extends State<_SearchTextField> {
   final TextEditingController controller = TextEditingController();
   final FocusNode focusNode = FocusNode();
 
+  @override
+  void initState() {
+    super.initState();
+    if (widget.ensureFocus) {
+      Future.delayed(const Duration(milliseconds: 200), () {
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/shared/icon_emoji_picker/icon_search_bar.dart:117]: Please add comments to explain this delay logic.

File: frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/emoji_icon_widget.dart (hunk 1 above)
Comment (asjqkkkk) [reply frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/emoji_icon_widget.dart:87]: It seems that for the same size, icons are larger than emojis

Before I changed the size:
image

After I changed the size:
image

As for why it's 0.9, I referred to the code in flowy_infra_ui, perhaps I should add relevant comments
image

Comment (LucasXu0) [reply frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/emoji_icon_widget.dart:87]: Screenshot 2024-12-30 at 10 17 12

**Pull request merged**